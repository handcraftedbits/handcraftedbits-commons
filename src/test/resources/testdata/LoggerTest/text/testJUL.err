WARN  | main                 | c.h.c.api.log.LoggerTest  | warning from JUL 
WARN  | main                 | c.h.c.api.log.LoggerTest  | warning from JUL with MDC | key1=value1, key2=value2
ERROR | main                 | c.h.c.api.log.LoggerTest  | severe from JUL 
ERROR | main                 | c.h.c.api.log.LoggerTest  | severe from JUL with MDC | key1=value1, key2=value2
