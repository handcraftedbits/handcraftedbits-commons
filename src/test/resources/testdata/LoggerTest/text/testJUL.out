TRACE | main                 | c.h.c.api.log.LoggerTest  | finest from JUL 
TRACE | main                 | c.h.c.api.log.LoggerTest  | finest from JUL with MDC | key1=value1, key2=value2
DEBUG | main                 | c.h.c.api.log.LoggerTest  | finer from JUL 
DEBUG | main                 | c.h.c.api.log.LoggerTest  | finer from JUL with MDC | key1=value1, key2=value2
DEBUG | main                 | c.h.c.api.log.LoggerTest  | fine from JUL 
DEBUG | main                 | c.h.c.api.log.LoggerTest  | fine from JUL with MDC | key1=value1, key2=value2
INFO  | main                 | c.h.c.api.log.LoggerTest  | info from JUL 
INFO  | main                 | c.h.c.api.log.LoggerTest  | info from JUL with MDC | key1=value1, key2=value2
